### DISCUSSION TITLE
Disable copilot without the pop up

### DISCUSSION THREAD

**Discussion by josiahsrc** [community/community#42959] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I frequently switch copilot off when I need to write my unique thoughts (without copilot interjecting). But I turn it back on right after. This works, but each time I enable/disable it, I get a second confirmation making sure that I want to proceed (which I always do).

<img width="603" alt="image" src="https://user-images.githubusercontent.com/37915005/210118770-968c8d15-c7e5-41ed-b70b-78d20d014f7c.png">

This becomes repetitive and slows down development. I've hooked up a hot-key (cmd+shift+c) to enable/disable copilot, but it's still not fast enough because I have to click the pop up. I'd really like to see an option to disable this pop-up.

**Comment by iamcjwilliams** [comment]
@josiahsrc Please visit here to trouble your copilot issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Reply by josiahsrc** [reply]
This isn't necessarily an issue, more of a feature request. Are you saying that this feature already exists?

**Reply by iamcjwilliams** [reply]
@josiahsrc the steps were given to troubleshoot the popup, do you have the latest version of the extension?

**Reply by josiahsrc** [reply]
Are you saying there's a way to remove the pop-up? I don't see that in the link provided. I have the latest version of the extension.

**Reply by iamcjwilliams** [reply]
What IDE are you using?

**Reply by josiahsrc** [reply]
Vscode

**Reply by iamcjwilliams** [reply]
@josiahsrc Currently, the ability to disable popups is not available, I will take this down as a feature request for the copilot team

**Reply by josiahsrc** [reply]
Awesome. Thanks!
